loadtest/loadtest_translations_1s_latency.py

```diff
@@ -0,0 +1,60 @@
+import logging
```

Comment:
Leaving a comment as a reminder that I plan to do a couple of load test runs before merging